README.md

```diff
@@ -43,10 +45,10 @@ crate-type = ["cdylib"]
 The bindings are currently generated from the API description of Godot 3.2 by default. To use the bindings with another version or a custom build, one currently has to use the bindings as a local dependency:
```

Comment:
I guess this can be changed too as full local clone is technically not needed anymore? With a section in the book about custom bindings this should be obsolete?

Reply:
Opened a PR to add a guide to the book: https://github.com/godot-rust/book/pull/7